test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -342,6 +342,12 @@ baz
             $expected[1] = $thirdLine
             Compare-Object -ReferenceObject $expected -DifferenceObject $result | Should -BeNullOrEmpty
         }
+        It "Should return first three lines at a time for -TotalCount 3 and -ReadCount 0" {
```

Comment:
The name should be more generic (than using 3).

Reply:
What about "Should return the same number of first lines which set in -TotalCount at one time for -ReadCount 0" Or "Should return first n lines at a time for -TotalCount <n> and -ReadCount 0" looks better?